Mods/ModVarietyPack/Defs/ThingDefs/Buildings_Furniture.xml

```diff
@@ -661,77 +661,11 @@
         <glowRadius>12</glowRadius>
         <glowColor>(217,217,208,0)</glowColor>
       </li>
-    </comps>
-    <researchPrerequisite>PowerI</researchPrerequisite>
-  </ThingDef>
-
-  <ThingDef ParentName="StandingLampBase">
-    <defName>StandingLamp_Red</defName>
-    <label>standing lamp (red)</label>
```

Comment:
There is a problem with deleting the colored standing lamps. 
As these are vanilla items, if you delete them, the defs here, they get added back to the furniture category with the vanilla textures. 
So at least the standing lamps must be kept.